data/resources/Urban_Jungle/_main.cfg

```diff
@@ -0,0 +1,31 @@
+#textdomain wesnoth-campaign-resources
+[textdomain]
+    name="wesnoth-campaign-resources"
+[/textdomain]
+
+#
+# This is a resource intended for use in select mainline campaigns, and should not be relied upon for UMC.
+# It may change in ways that aren't backwards-compatible with or that negatively affect the balance of UMC content that uses it.
+#
+[resource]
+    id=campaignResource_urban_jungle
+[/resource]
```

Comment:
You should add a `define=` to this tag so that the contents of the resource are only in scope when the resource is loaded.

Reply:
Thanks, done. That makes sense; or else the contents will be in scope everywhere, since `{resource}` is now loaded in the toplevel `_main.cfg`